libs/api/__tests__/specs/peer.test.ts

```diff
@@ -150,7 +150,6 @@ describe('Peers', () => {
     test('can update peer profile', async () => {
       const input: PeerProfileInput = {
         name: 'test peer profile',
-        logoID: 'logoID123',
```

Comment:
This was causing an error and the snapshot included it. We don't have any image uploads in tests so far so I deleted it for now.